src/stories/DataResultItem.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Meta } from '@storybook/react/types-6-0';
+import { Story } from '@storybook/react';
+import DataResultItem, { DataResultItemProps } from '../components/molecules/DataResultItem';
+
+export default {
+    title: 'DataExplorer/DataExplorer/Item',
+    component: DataResultItem,
+} as Meta;
+
+const Template: Story<DataResultItemProps> = (args) => <DataResultItem {...args} />;
+
+export const Primary = Template.bind({});
+Primary.args = {
+    title: 'This is an title',
+    description:
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.',
+    tags: ['Energi', 'Miljø', 'Helse', 'Pils', 'Raketter'],
```

Comment:
The tags should be able to edited/controlled without typing in a string, I would recommend looking into the multiselect control:D https://storybook.js.org/docs/react/essentials/controls#annotation

Reply:
Also one should be able to choose the visualisation type of an item from a predefined set of types, this could be "controlled" with a select control